es-models/case_centric/case_centric.mapping.yaml

```diff
@@ -6,6 +6,47 @@ _source:
   - gene.biotype
   - gene.is_cancer_gene_census
 properties:
+  aliquot_ids:
+    type: keyword
+  analyte_ids:
+    type: keyword
```

Comment:
Do we need the normalizer definitions found in the case mappings here as well?

Reply:
I hadn't seen them added anywhere for our previous work so I didn't carry them over. @BenX00 Do you have any thoughts on this?

Reply:
Thank you for sending the Jira ticket associated with this functionality @kulgan: https://jira.opensciencedatacloud.org/browse/DAT-2302

From this I found the following: "The cDAVE feature mainly relies on the case_centric mutation index. As the mutation indexer pulls the case data from the gdc_from_graph index when building the mutation indices, normalizing casing in the graph index should also normalize the casing in the mutation indices without any additional changes required."

This leads me to believe we don't need to include these from an indexing standpoint. Offline you mentioned this may come into play at query time so we still need to determine if that will impact this change.

Reply:
This was discussed offline and the normalizer is not needed in the case_centric properties.